src/content/get-started/dev-quickstart.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Using Okteto as a Developer
+description: This guide will help you to get started with Okteto as a developer
+sidebar_label: Quickstart
+id: dev-quickstart
+pagination_prev: get-started/install/index
+---
+
+## Introduction
+
+Welcome to the Okteto Quick Start Guide! This guide will help you, as a developer, get started with Okteto in a few steps. Once your platform team has set up Okteto and configured your account, you can follow this guide to deploy your Development Environments and start coding in no time.
+
+<iframe
+  width="560"
+  height="315"
+  src="https://www.youtube.com/embed/eEJWJObxM2I?si=Q8MV1MEQ1WZa93YQ"
+  title="Okteto Developer Quickstart Guide"
+  frameborder="0"
+  allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share"
+  referrerpolicy="strict-origin-when-cross-origin"
+  allowfullscreen
+></iframe>
+
+## Step 1: Sign In to Okteto
+
+1. Open your web browser and navigate to your Okteto instance URL. It should look something like `https://okteto.mycompany.com`
```

Comment:
In all other tutorials, we are using https://okteto.example.com. Happy to change it, but we should be consistent across docs

Reply:
I feel like for these onboardings we should use something like `mycompany` just so they know that the url is their own

Reply:
That's the same for other tutorials. Whatever we pick, we should be consistent